src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -1,5 +1,11 @@
 {
     "SAMPLE_CONFIG_DB_JSON": {
+        "VRF": {
+                "default": {
+                },
+                "Vrf_blue": {
+                }
+        },
```

Comment:
Include sample for PORTCHANNEL INTERFACE LIST

Reply:
Better to include both configs as below:
```
"PortChannel0004": {},

and 
"PortChannel0004": { vrf_name: 'Vrf_xxx'},
```

Reply:
Addressed